contracts/erc20-gateway/ERC20GatewayBase.sol

```diff
@@ -39,6 +39,11 @@ contract ERC20GatewayBase {
         "WithdrawIntent(address valueToken,address utilityToken,uint256 amount,address beneficiary)"
     );
 
+    /* Storage */
```

Comment:
Add two new lines before top level declarations.
Do read the guideline: https://github.com/mosaicdao/developer-guidelines/blob/master/SOLIDITY_STYLE_GUIDE.md#blank-lines

Reply:
Understood and changed.